packages/updating-element/src/updating-element.ts

```diff
@@ -544,6 +558,10 @@ export abstract class UpdatingElement extends HTMLElement {
     this.requestUpdate();
   }
 
+  addController(controller: Controller) {
```

Comment:
I still like `addCallbacks` but won't insist.

Reply:
Going with `addController` as that's consensus.